lib/database_connection.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @con = PG.connect(dbname: dbname)
+  end
+
+  def self.query(sql) 
+    @con.exec(sql)
+  end
+end
```

Comment:
Neat that this refactors the connection call :) I haven't seen that before. 

However, when I ran the code on my local machine, localhost threw an error when it tried to call .exec (for the .query method).  It said @con was nil:NilClass. As I'm not familiar with these class methods, I can't tell whether the bug was in my setup or in the code. I'm guessing this worked on your machine though, so likely I haven't replicated your setup somehow.

I can show you in our zoom call later if you like :)